src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -664,7 +664,10 @@ export class BrowsingContextImpl {
         break;
     }
 
-    return {};
+    return {
+      navigation: this.loaderId ?? null,
```

Comment:
I think it should get a new ID according to the spec. Is there a WPT change?

Note that reload is not a same-document navigation.

Reply:
Where would the ID come from? CDP `Page.reload` doesn't return any ID.

Reply:
it might come via lifecycle events or we change Page.reload implementation

Reply:
If it comes via lifecycle events, then the getters should be correct. No?

Reply:
idk, how about a test for that?

Reply:
Augmented E2E tests. This seems to confirm my hypothesis. The tests are all passing.

`None` is the only case wherein we do not have the correct (new) ID.

Reply:
what if a reload is triggering a redirect?

Reply:
Can we defer this question to the upstream issue and to WPT tests?

Reply:
https://github.com/w3c/webdriver-bidi/issues/532